tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "declaration": true,
     "strict": true
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src/**/*.ts", "src/*.ts"],
+  "exclude": ["node_modules"],
+  "sourceRoot": "./src",
```

Comment:
Seems to be a `compilerOptions`, so it's on the wrong place here. But why would it be needed anyways?

Reply:
That was troubleshooting stuff should not be needed